comp/core/gui/guiimpl/gui.go

```diff
@@ -240,49 +229,57 @@ func serveAssets(w http.ResponseWriter, req *http.Request) {
 	w.Write(data)
 }
 
+func (g *gui) getAccessToken(w http.ResponseWriter, r *http.Request) {
+
+	intentToken := r.URL.Query().Get("intent")
+	// authToken is present in the query when the GUI is opened from the CLI
+	if intentToken == "" {
+		w.WriteHeader(http.StatusUnauthorized)
+		http.Error(w, "missing intentToken", 401)
+		return
+	}
+	if _, ok := g.intentTokens[intentToken]; !ok {
+		w.WriteHeader(http.StatusUnauthorized)
+		http.Error(w, "invalid intentToken", 401)
+		return
+	}
+
+	// Remove single use token from map
+	delete(g.intentTokens, intentToken)
+
+	// generate accessToken
+	accessToken := g.auth.GenerateAccessToken()
+
+	// accessToken, err := g.auth.Authenticate(intentToken)
```

Comment:
Should we remove this line?

Reply:
Done in 269022d16e493c88bdcdb13579f4cf583e2e6031.